src/data/global.ts

```diff
@@ -102,7 +120,13 @@ export interface Address {
    *
    * @see https://docs.mollie.com/overview/common-data-types?path=postalCode#address-object
    */
-  postalCode: string;
+  postalCode?: string;
+  /**
+   * The customer's email address.
+   *
+   * @see https://docs.mollie.com/overview/common-data-types?path=email#address-object
+   */
+  email?: string;
```

Comment:
@fjbender `email` is missing from the [common type definition](https://docs.mollie.com/reference/common-data-types?path=email#address-object), but I think that's a mistake, as it is included where the address is used (payments/orders)

Reply:
It's likely a documentation bug as the common type definition is not auto-generated.